helm/opendistro-es/values.yaml

```diff
@@ -12,7 +12,7 @@
 # permissions and limitations under the License.
 
 certmanager:
-  enabled: false
+  enabled: true
```

Comment:
Please do the cert-manager changes in a separate commit if they are necessary for the users, or create a separate PR if they can be done separately.